optimum/onnxruntime/modeling_ort.py

```diff
@@ -90,12 +90,19 @@ class ORTModel(OptimizedModel):
     base_model_prefix = "onnx_model"
     auto_model_class = AutoModel
 
-    def __init__(self, model=None, config=None, **kwargs):
+    def __init__(self, model: ort.InferenceSession, config, **kwargs):
```

Comment:
Because to me `model` and `config` should be required arguments, it does not make sense to initialize an `ORTModel` without an InferenceSesson provided. What do you think?

Reply:
Lets stick with what we are doing everywhere with named args